.travis.yml

```diff
@@ -24,3 +25,6 @@ jobs:
     - stage: Integration Tests
       name: "Browser Tests"
       script: make run_browser_tests
+
+after_success:
```

Comment:
`after_success` isn't good because it gets executed in all build variants, and we want only one execution

Reply:
needs to use `deploy` instead https://docs.travis-ci.com/user/deployment/script/

Reply:
may need [`skip_cleanup`](https://docs.travis-ci.com/user/deployment#uploading-files-and-skip_cleanup)

Reply:
the `Makefile` has empty `push_*` targets, but that would push the image(s) before the `Integration Tests` stage is run

Reply:
though that stage is empty?

Reply:
so in the end this should be moved to the `push_server_container` target (really named `push_server_image`)

Reply:
There's already a placeholder step in the `Makefile` which is run per service